operations/experimental/perm-service.yml

```diff
@@ -177,26 +183,18 @@
           port: 8443
           ca_certs:
           - ((uaa_ca.certificate))
-
         cloud_controller:
           hostname: api.((system_domain))
           port: 443
           client_id: cloud_controller_monitor
           client_secret: ((perm_uaa_clients_cloud_controller_monitor_secret))
           client_scopes:
           - cloud_controller.admin_read_only
-
+          ca_certs:
+          - ((router_ca.ca))
```

Comment:
This should be `router_ca.certificate`. It is the same value as `router_ca.ca` but for consistency we'd like to reference all CAs in the same manner.